db/seeds.rb

```diff
@@ -5,3 +5,24 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Destination.create(continent: "North America", description: "North American continent description.",
+                    lat: 41.8369, long: -87.6847, image_url: "glacier.jpg" )
+Destination.create(continent: "South America", description: "South American continent description.",
+                    lat: -42.8369, long: -17.6847, image_url: "Chile.jpg" )
+Destination.create(continent: "Europe", description: "European continent description.",
+                    lat: 21.8369, long: -47.6847, image_url: "dolomites.jpg" )
+Destination.create(continent: "Asia", description: "Asian continent description.",
+                    lat: 31.8369, long: -77.6847, image_url: "Pakistan.jpg" )
+Destination.create(continent: "Africa", description: "African continent description.",
+                    lat: 21.8369, long: -67.6847, image_url: "Simian.jpg" )
+Destination.create(continent: "Oceania", description: "Oceanian continent description.",
+                    lat: -45.8369, long: -57.6847, image_url: "NewMilford.jpg" )
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside parentheses detected.